Setup.sh

```diff
@@ -18,9 +18,8 @@ BINARIES_DIR="$(pwd)/SpatialGDK/Binaries/ThirdParty/Improbable"
 SCHEMA_COPY_DIR="$(pwd)/../../../spatial/schema/unreal/gdk"
 SCHEMA_STD_COPY_DIR="$(pwd)/../../../spatial/build/dependencies/schema/standard_library"
 SPATIAL_DIR="$(pwd)/../../../spatial"
-DOMAIN_ENVIRONMENT_VAR=
-if [[ "$*" == "--china" ]]; then
-    DOMAIN_ENVIRONMENT_VAR=--domain spatialoschina.com --environment cn-production
+if [[ "${1:-}" == "--china" ]]; then
```

Comment:
"${1:-}" - does this only search the first argument? If so, should we be searching all args because the order could be reversed if more arguments are added?

Reply:
yes, this checks the first argument and if there is none it will be equal to "" 
In the future, if we add more arguments, it is something that we should take a look at. but as we have only one valid argument right now, I think it is fine to simply check the first one.